EastAgile/twitterbot
# [b] Remove manual handling of Twitter character limit error due to in accuracy

Description
--
- Twitter count character limit AFTER shorten URLs, hence our inaccuracy
- Remove redundant constants & errors

How to test
--
- Send a tweet with an URL that in total (text + URL) exceeds 140 characters but the text alone is less than 140 characters (~100 characters is fine). An APIError should NOT be raised.
- Send a tweet with text alone exceeds 140 characters. An APIError should be raised.
